interface/ceed-operator.c

```diff
@@ -248,6 +252,50 @@ int CeedCompositeOperatorAddSub(CeedOperator compositeop, CeedOperator subop) {
   return 0;
 }
 
+/**
+  @brief Duplicate a CeedOperator with a refrence Ceed to fallback for advanced
+           CeedOperator functionality
+
+  @param op           CeedOperator to create fallback for
+
+  @return An error code: 0 - success, otherwise - failure
+
+  @ref Developer
+**/
+int CeedOperatorCreateFallback(CeedOperator op) {
+  int ierr;
+
+  // Fallback Ceed
+  const char *resource;
+  ierr = CeedOperatorGetFallbackResource(op, &resource);
+  Ceed ceedref;
+  ierr = CeedInit(resource, &ceedref); CeedChk(ierr);
+  op->fallbackceed = ceedref;
+
+  // Clone Op
+  CeedOperator opref;
+  ierr = CeedCalloc(1, &opref); CeedChk(ierr);
+  memcpy(opref, op, sizeof(*opref)); CeedChk(ierr);
+  opref->data = NULL;
+  opref->setupdone = 0;
+  opref->ceed = ceedref;
+  ierr = ceedref->OperatorCreate(opref); CeedChk(ierr);
+  opref->parentceed = op->ceed;
+  op->fallback = opref;
```

Comment:
Are we making hanging references above? Is it possible to infinite loop?

Reply:
I don't believe its possible to infinite loop as written. It runs Valgrind clean when I build with only the CPU backends and use a fallback rather than the `/cpu/self/ref/blocked` impl of `AssembleLinearQF`.

Reply:
What if the delegate doesn't implement the desired method and attempts to delegate to itself? This may not be possible at present, but I'd be concerned about it happening in the future.

Reply:
I put in a check that a ceed cannot delegate to itself.